daemon/src/routes_maker.rs

```diff
@@ -112,14 +111,29 @@ pub async fn post_sell_order(
     maker: &State<Maker>,
     _auth: Authenticated,
 ) -> Result<(), HttpApiProblem> {
+    let funding_rate = match order.funding_rate {
+        None => None,
+        Some(funding_rate) => Some(FundingRate::new(funding_rate).map_err(|e| {
+            HttpApiProblem::new(StatusCode::BAD_REQUEST)
+                .title("Invalid funding rate")
+                .detail(format!("{e:#}"))
+        })?),
```

Comment:
Can we do the expression with the `?` on a new line please? This took a while to parse in my head :D